python/graphstorm/gpartition/random_partition.py

```diff
@@ -0,0 +1,73 @@
+"""
+    Copyright 2023 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Random partition algorithm
+"""
+import os
+import logging
+import json
+from typing import Dict, List
+
+import numpy as np
+import pyarrow as pa
+import pyarrow.csv as pa_csv
+
+from .local_partition_graph import LocalPartitionAlgorithm
```

Comment:
where is local_partition_graph?

Reply:
Sorry file was not added to the commit, I added it now.